pkg/resource/secret.go

```diff
@@ -41,7 +41,7 @@ func (gs *generatorSecret) GetNamespace() string {
 }
 
 func (gs *generatorSecret) GetName() string {
-	return gs.name
+	return "image-registry"
```

Comment:
This seems due for being turned into a constant.

Reply:
updated in next push